modules/ocf_ceph/manifests/osd.pp

```diff
@@ -0,0 +1,33 @@
+# Ceph object store daemon configuration
+# Set ocf_ceph::osd::disks to be the disks allocated to ceph
+# If you have issues just dd if=/dev/zero the disk and try again
```

Comment:
What issues did you see here? Worth mentioning

Reply:
Agreed, and zeroing disks is perhaps a dangerous solution to suggest here without enough context